google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/spi/v1/PublisherImpl.java

```diff
@@ -150,17 +140,21 @@ public PublisherStats getStats() {
 
   @Override
   public Duration getMaxBundleDuration() {
-    return maxBundleDuration;
+    return bundlingSettings.getDelayThreshold();
   }
 
   @Override
   public long getMaxBundleBytes() {
-    return maxBundleBytes;
+    return bundlingSettings.getRequestByteThreshold();
+  }
+
+  private boolean hasBundlingBytes() {
+    return getMaxBundleBytes() > 0;
   }
 
   @Override
   public long getMaxBundleMessages() {
```

Comment:
I'd like to defer this for when we make Publisher a concrete class. Then we'll have fewer places to change.